apps/mongodb.go

```diff
@@ -0,0 +1,291 @@
+// Copyright 2021 Google LLC
```

Comment:
Can we add a MD file (aka documentation) as well? https://github.com/GoogleCloudPlatform/ops-agent/tree/master/docs

Reply:
Thanks! Looks like the tests are not executed though: This file is missing Mongodb? https://github.com/GoogleCloudPlatform/ops-agent/blob/051df889b02eba0d08e0087dd1ebfc8bb05acf25/integration_test/third_party_apps_data/agent/ops-agent/linux/supported_applications.txt